src/kakuro/GameController.java

```diff
@@ -11,7 +11,7 @@
 {
     public GameView view;
     public GameModel model;
-    private Boolean gui = true;
+    private Boolean gui = false;
```

Comment:
pretty sure we want true by default in any cases it is false by default without initialization